debian/rules

```diff
@@ -4,4 +4,5 @@
 	dh $@ --with=systemd,python2
 
 override_dh_installinit:
-	dh_installinit -r --no-start -- start 15 2 .
+	dh_installinit --name=inithooks -r --no-start
+	dh_installinit --name=inithooks-lxc
```

Comment:
@lamby - any hints on how to get this additional service (file: `inithooks.inithooks-lxc.server`; installs file `inithooks-lxc.service`) to be enabled by default?!?

It installs ok (i.e. file goes where it should etc), but isn't enabled (i.e. no symlink /etc/systemd/system/basic.target.wants/inithooks-lxc.service -> /lib/systemd/system/inithooks-lxc.service). 

(please note this PR is incomplete, but should be near enough for this convo).

FWIW on an installed system (note this isn't in an LXC build; but it exactly matches what I'm seeing on LXC too):
```
root@core ~# systemctl status inithooks.service 
* inithooks.service - inithooks: firstboot and everyboot initialization scripts
   Loaded: loaded (/lib/systemd/system/inithooks.service; enabled; vendor preset: enabled)
   Active: inactive (dead) since Tue 2018-05-15 07:03:02 UTC; 4min 27s ago
 Main PID: 563 (code=exited, status=0/SUCCESS)

May 15 06:56:43 core systemd[1]: Starting inithooks: firstboot and everyboot initialization scripts...
May 15 06:56:56 core passwd[3650]: password for 'root' changed by 'root'
May 15 07:02:56 core chpasswd[3763]: pam_unix(chpasswd:chauthtok): password changed for root
May 15 07:03:02 core systemd[1]: Started inithooks: firstboot and everyboot initialization scripts.
```
(works as expected)
```
root@core ~# systemctl status inithooks-lxc.service 
* inithooks-lxc.service - inithooks-lxc: firstboot and everyboot initialization scripts (lxc)
   Loaded: loaded (/lib/systemd/system/inithooks-lxc.service; disabled; vendor preset: enabled)
   Active: inactive (dead)
```
Note `disabled`! (Although also note `vendor preset: enabled`)
```
root@core ~# ls -l /etc/systemd/system/basic.target.wants/                        
total 0
lrwxrwxrwx 1 root root 37 May 15 05:47 inithooks.service -> /lib/systemd/system/inithooks.service
```
No symlink for `/lib/systemd/system/inithooks-lxc.service` :cry: 
But I can enable it manually ok?!
```
root@core ~# systemctl enable inithooks-lxc.service 
Created symlink /etc/systemd/system/basic.target.wants/inithooks-lxc.service -> /lib/systemd/system/inithooks-lxc.service.

root@core ~# systemctl status inithooks-lxc.service 
* inithooks-lxc.service - inithooks-lxc: firstboot and everyboot initialization scripts (lxc)
   Loaded: loaded (/lib/systemd/system/inithooks-lxc.service; enabled; vendor preset: enabled)
   Active: inactive (dead)
```

Worst case scenario, I could manually enable it as part of the LXC build process, but it's annoying that is doesn't "just work" like I think it should... 

Any input appreciated.

Reply:
@JedMeister:

I think you need to:

a)

Revert your renames of the `init` files.

b) Set:

```
override_dh_systemd_enable:
	dh_systemd_enable debian/*.service
```

otherwise it won't know to enable the other `.service` file

c) Fix the `WantedBy` target from `basic.target` to `multi-user.target`.

Unsure, just got off a flight :)

Reply:
Cheers mate. Really appreciate it. That'll give me some more to play with tomorrow. Hope you get a chance to rest up :)